LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Extensions/DbContextExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using LvivCompany.Bookstore.DataAccess;
+using LvivCompany.Bookstore.Entities;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace LvivCompany.Bookstore.Web
+{
+    public static class DbContextExtensions
+    {
+        public static IServiceCollection AddDbContexts(this IServiceCollection services, IHostingEnvironment env, IConfiguration config)
+        {
+            if (env.IsDevelopment())
```

Comment:
No need to check env here unless you want to use some other provider (e.g. InMemory). Just add two db contexts here. Use `config.GetConnectionString` in both cases